sources/executor/execute_commands/execute_wait.c

```diff
@@ -6,25 +6,17 @@
 /*   By: quentinbeukelman <quentinbeukelman@stud      +#+                     */
 /*                                                   +#+                      */
 /*   Created: 2024/04/22 21:17:34 by quentinbeuk   #+#    #+#                 */
-/*   Updated: 2024/04/23 21:05:11 by quentinbeuk   ########   odam.nl         */
+/*   Updated: 2024/04/24 19:19:49 by qtrinh        ########   odam.nl         */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "../../../includes/minishell.h"
 
-static int	handle_signal(int signal)
```

Comment:
Keep this function. I am still working on the exit codes here.

Reply:
my bad! My opinion is that the signals are handled correctly in the childprocess and I dont believe it's neccesary to handle the signals at waitpid function. will revert it